includes/Block/Web_Stories_Block.php

```diff
@@ -84,6 +85,11 @@ public function init() {
 			$this->get_script_settings()
 		);
 
+		// Check to see if block is already registered.
+		if ( $this->is_register( self::BLOCK_NAME ) ) {
+			return;
+		}
+
```

Comment:
Why is this needed?

Reply:
> Check is added to see if block is already registered, as in activate it is registered twice.

It because the services are register again on activation.